3.3-nginx/docker-entrypoint.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash -e
 
+for file in "etc/nginx/nginx.conf" "etc/nginx/conf.d/default.conf" "etc/modsecurity.d/modsecurity-override.conf"
+do
+    envsubst "$(awk 'BEGIN{for(v in ENVIRON) print "$"v}')" <$file | sponge $file
```

Comment:
This will bite us one day.

Can we make this code less cryptic?

Reply:
What do you mean by less cryptic? The previous version was without for loop.